presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -20,8 +20,11 @@
 import io.airlift.units.Duration;
```

Comment:
Please improve commit message. You not only round decimal. In general you added support for decimal values with unspecified parameters. Please make it clear in the commit message. Rounding is just a part of it.